zh.js

```diff
@@ -1,747 +1,747 @@
+settings.editing.progm.shadow 蓄力条阴影
+settings.editing.progmOffX 蓄力条水平位置
+settings.editing.progmOffY 力条垂直位置
+settings.editing.progmRot 蓄力条角度
+settings.editing.saturation 饱和度 (游戏中)
+settings.editing.menuSaturation 饱和度 (用户界面)
+settings.editing.hue 色调 (游戏中)
+settings.editing.menuHue 色调 (用户界面)
+settings.editing.vignette 晕影效果（使屏幕边缘颜色变暗)
+settings.editing.feedLimit 击杀列表长度限制
+settings.editing.depth 深度图模式
+settings.editing.green 绿幕模式
+settings.editing.tracercol 弹道轨迹
+settings.editing.tracer 弹道轨迹
+settings.editing.tracer.offset 弹道显示位置
+settings.shaders.header 着色器输入
+settings.mods.header 自定义
+settings.mods.load 加载模组
+settings.mods.logo 允许模组变更界面图标
+settings.mods.auto 自动加载模组
+settings.mods.borders 狙击镜边界
+settings.mods.borderCol 狙击镜边界
+settings.mods.borderOpac 狙击镜边界透明度
+settings.mods.scope 狙击镜准心图标
+settings.mods.scope.paste 狙击镜准心图标网址
+settings.mods.scope.opacity 狙击镜准心透明度
+settings.mods.scope.width 狙击镜准心宽度
+settings.mods.scope.height 狙击镜准心高度
+settings.mods.dot 准星图标
+settings.mods.dot.width 准星图标宽度
+settings.mods.dot.height 准星图标高度
+settings.mods.dot.paste 准星图标网址
+settings.mods.endmessage 比赛结束后自动消息
+settings.mods.profile 用户图标
+settings.mods.profile.paste 用户图标网址
+settings.mods.ammo 弹药图标
+settings.mods.ammo.paste 弹药图标网址
+settings.mods.kills 击杀图标
+settings.mods.kills.paste 击杀图标网址
+settings.mods.deaths 死亡图标
+settings.mods.deaths.paste 死亡图标网址
+settings.mods.streak 连杀图标
+settings.mods.streak.paste 连杀图标网址
+settings.mods.timer 时钟图标
+settings.mods.timer.paste 时钟图标网址
+settings.mods.usedamage 覆盖受伤屏幕效果
+settings.mods.damage 受伤屏幕效果图标
+settings.mods.damage.paste 受伤屏幕效果网址
+settings.mods.overlay 游戏结束图标
+settings.mods.overlay.paste 游戏叠层图片网址
+settings.social.region 各地区统计数据
+windows.settings.header 游戏设定
+windows.mail.header Krunker邮箱
+windows.servers.header 服务器
+windows.servers.none 无可加入游戏...
+windows.servers.host + 创建游戏
+windows.servers.hide 隐藏已满的房间
+windows.servers.priority 优先区域
+windows.servers.online {0} 人正在游戏中
+windows.servers.social {0} 正在游戏社交网站浏览中
+windows.loadout.header 更换角色
+windows.loadout.class 角色
+windows.loadout.primary 主武器
+windows.loadout.primary.attach 配件
+windows.loadout.primary.reticle 准星
+windows.loadout.primary.scope 狙击镜准心
+windows.loadout.secondary 副武器
+windows.loadout.secondary.skin 副武器皮肤
+windows.loadout.melee 近战武器
+windows.loadout.hat 头饰
+windows.loadout.back 背部饰品
+windows.loadout.waist 腰部饰品
+windows.loadout.shoe Shoes 鞋子
+windows.loadout.face 表情
+windows.loadout.spray 涂鸦
+windows.loadout.color.skin 皮肤颜色
+windows.loadout.color.chat 聊天框中玩家用户名颜色
+windows.loadout.color.hair 头发颜色
+windows.loadout.dye 服装
+windows.loadout.pet 宠物
+windows.loadout.stat 击杀统计
+windows.loadout.flag 玩家国家旗帜
+windows.mods.drop 将模组文件放在此处或单击
+windows.mods.paste 粘贴模组网址
+windows.mods.load 加载模组
+windows.mods.none 没有找到任何模组
+windows.mods.header 模组管理器
+windows.mods.guide 模组指南
+windows.mods.list 模组列表
+windows.mods.load.header 加载模组
+windows.mods.publish 发布模组
+windows.mods.viewer 观看者
+windows.account.header 帐号
+windows.account.recover 联系 <a>recovery@yendis.ch</a> 找回账号密码

```

Comment:
https://github.com/mty22/krunker.io-translations/blob/main/en.js#L425

need to check what is set across all files